src/platform/packages/private/kbn-grid-layout/grid/grid_row/grid_row.tsx

```diff
@@ -98,54 +95,62 @@ export const GridRow = React.memo(({ rowIndex }: GridRowProps) => {
        * reasons (screen readers and focus management).
        */
       const gridLayoutSubscription = gridLayoutStateManager.gridLayout$.subscribe((gridLayout) => {
+        if (!gridLayout[rowIndex]) return;
         const newPanelIdsInOrder = getKeysInOrder(gridLayout[rowIndex].panels);
         if (panelIdsInOrder.join() !== newPanelIdsInOrder.join()) {
           setPanelIdsInOrder(newPanelIdsInOrder);
         }
       });
 
-      const columnCountSubscription = gridLayoutStateManager.runtimeSettings$
```

Comment:
Moved this subscription to the other CSS variables subscription in `useGridLayoutState` - we shouldn't have a case where each section has a different grid column count, so there is no reason to duplicate this. Oops!